gtsfm/scene_optimizer.py

```diff
@@ -213,12 +211,20 @@ def create_computation_graph(
         if self._save_bal_files:
             # save the input to Bundle Adjustment (from data association)
             auxiliary_graph_list.append(
-                dask.delayed(write_sfmdata_to_disk)(ba_input_graph, os.path.join(RESULTS_PATH, "ba_input.bal"))
+                dask.delayed(io_utils.write_cameras)(ba_input_graph, os.path.join(RESULTS_PATH, "ba_input_cameras.txt"))
```

Comment:
i see why you named these differently, because we have multiple versions, and we want to differentiate between them.

But can we just put them in separate subdirectories? Per https://colmap.github.io/cli.html, we need to preserve the names to be compatible:

COLMAP as far as I know 
<img width="615" alt="Screen Shot 2021-03-22 at 10 42 53 PM" src="https://user-images.githubusercontent.com/16724970/112084899-f2682e80-8b5f-11eb-862d-f3f053d0fd87.png">